e2e/05-payments-receive-flow.e2e.spec.ts

```diff
@@ -27,7 +26,9 @@ describe("Receive BTC Amount Payment Flow", () => {
   })
 
   it("Click Request Specific Amount", async () => {
```

Comment:
we should certainly also tap on paycode and onchain and verify the behavior (ie: a QR code is showing up)